ehr/src/org/labkey/ehr/EHRServiceImpl.java

```diff
@@ -456,7 +456,6 @@ public void registerOptionalClinicalHistoryResources(Module module)
         EHRService.get().registerLabworkType(new AntibioticSensitivityLabworkType(module));
         EHRService.get().registerLabworkType(new ChemistryLabworkType(module));
         EHRService.get().registerLabworkType(new HematologyLabworkType(module));
-        EHRService.get().registerLabworkType(new iStatLabworkType(module));
```

Comment:
To remove this here, we need to add it to WNPRC_EHRModule.doStartupAfterSpringConfig() so they don't lose it.

Reply:
done